scripts/trigger-buildkite-pipeline.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
```

Comment:
@michaelpj  now that I extracted it i agree that it's nicer to have this in a separate file/script :+1: